base.c

```diff
@@ -168,8 +168,27 @@ int isOneByteOp() {
         (token >= ADD && token <= MODULO);
 }
 
-int isOp() {
-  return isOneByteOp();
+int isKonstantOp(){
+    //TODO: implement isKonstantOp()
+    return false;
 }
 
+int isVarOp(){
+    //TODO: implement isVarOp()
+    return false;
+}
 
+int isOp() {
+  if(isOneByteOp()){
+      return 0;
+  }
+  if(isKonstantOp()){
+      return 2;
+  }
+  if(isVarOp()){
+      return 1;
+  }
+  else{
```

Comment:
nope, since all the ifs have returns on them it's functionally the same thing

if something return ends the chain, if it's false it will go to the next if, it will only reach the else if all the if's are false